docs/index.md

```diff
@@ -19,14 +19,16 @@ yarn add banano vee-validate
 # If using the tailwind plugin
 yarn add @headlessui/tailwindcss @tailwindcss/forms
 ```
-
+Then in some component you can import banano's components as shown belows:
```

Comment:
For clarity, I'd change this sentence to something like:
`Then, in any component, you can import components from Banano as shown below:`